apps/coscrad-frontend/src/utils/generic-components/presenters/detail-views/resource-detail-thumbnail-presenter.tsx

```diff
@@ -30,6 +31,7 @@ export const ResourceDetailThumbnailPresenter = ({
     <Card>
         <CardContent>
             <Grid container spacing={1} columns={{ xs: 2, sm: 4, md: 12 }}>
+                <TestIdDiv testId={id} />
                 <Grid item xs={2} sm={1} md={2}>
                     {/* Preview will eventually include images taken from video or photos, etc. */}
                     <ResourcePreviewIconFactory resourceType={type} size="md" />
```

Comment:
In the case of Bibliographic References this could use the specific reference type icon instead of the `ResourceType` icon